src/query/procedure/module.cpp

```diff
@@ -320,6 +322,368 @@ void RegisterMgTransformations(const std::map<std::string, std::unique_ptr<Modul
   module->AddProcedure("transformations", std::move(procedures));
 }
 
+namespace {
+bool IsAllowedExtension(const auto &extension) {
+  constexpr std::array<std::string_view, 1> allowed_extensions{".py"};
+  return std::any_of(allowed_extensions.begin(), allowed_extensions.end(),
```

Comment:
I was about to write `std::find` but I really like the idea of using `std::any_of` to return a bool. Really nice :fire: